sims/verilator/Makefile

```diff
@@ -59,7 +59,8 @@ VERILATOR_NONCC_OPTS = \
 	+define+PRINTF_COND=\$$c\(\"verbose\",\"\&\&\"\,\"done_reset\"\) \
 	+define+STOP_COND=\$$c\(\"done_reset\"\) \
 	--assert \
-	--output-split 20000 \
+        --output-split 10000 \
```

Comment:
Nit: This is off by a space. Also is there any intuition behind these numbers? Or is it just arbitrary?

Reply:
`output-split` roughly specifies how much to split the simulator cpp into different files (lower number means more files)

`output-split-cfunc` specifies how aggressively to split individual c functions into different functions (lower number means more functions)

Reply:
I'm assuming you just tested this with the `MegaBoomConfig` and chose random numbers to see if it worked?

Reply:
I actually tested with LargeBoom, but yeah you can do some fine tuning to improve it even further, but I don't see the need for that.